spec/ImageDiffSpec.js

```diff
@@ -313,13 +313,28 @@ describe('ImageUtils', function() {
 
       it('should calculate difference', function () {
         a = imagediff.createImageData(1, 1),
+        a.data[1] = 200,
```

Comment:
semicolon instead of a comma at the end of the line? I haven't come across this syntax before.